packages/client/wallets/aa/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "@crossmint/client-sdk-aa",
+    "version": "0.0.9-alpha.6",
```

Comment:
Is this version seen externally? can we keep it in sync w other packages?

Reply:
in sync, other repo should be archived after this
https://www.npmjs.com/package/@crossmint/client-sdk-aa

Reply:
i mean in sync w our client UI ones